src/main/java/team/themoment/hellogsmv3/domain/auth/controller/AuthenticationController.java

```diff
@@ -26,4 +28,11 @@ public BasicAuthenticationDto getAuthenticationInfo(
     public BasicAuthenticationDto getMyAuthenticationInfo() {
         return queryAuthenticationById.execute(manager.getId());
     }
+
+    @DeleteMapping("/authentication/me")
+    public CommonApiMessageResponse deleteMyAuthentication() {
+        Long authenticationId = manager.getId();
+        deleteMyAuthenticationService.execute(authenticationId);
```

Comment:
```suggestion
        deleteMyAuthenticationService.execute(manager.getId());
```
인라인화 시키면 더 깔끔해질것 같습니다!

Reply:
반영하였습니다.
[79d47e6](https://github.com/themoment-team/hellogsm-server-v3/pull/36/commits/79d47e61e3d4b9951f6f0166ee8f70fdbfcc2ef3)